### ISSUE TITLE
Bad Request for Host a remote evaluation challenge 

### ISSUE THREAD

**Issue by Espere-1119-Song** [Cloud-CV/EvalAI-Starters#87]
I am holding a remote evaluation challenge with EvalAI.

I can succcessfully download the submissions from the website.

However, when I need to update the leaderboard with `update_finished`, I meet error in `update_submission_data.` I found that when goes through it goes wrong in function `make_request` and `response.raise_for_status()`. Specifically:

```
response = requests.request(
                method=method, url=url, headers=headers, data=data
            )
```
It reports that `requests.exceptions.HTTPError: 400 Client Error: Bad Request for url: https://eval.ai/api/jobs/challenge/2253/update_submission/` and the content in the website is `{"detail":"Authentication credentials were not provided."}`

How can I fix it?

**Comment by Liuziyu77** [comment]
> I am holding a remote evaluation challenge with EvalAI.
> 
> I can succcessfully download the submissions from the website.
> 
> However, when I need to update the leaderboard with `update_finished`, I meet error in `update_submission_data.` I found that when goes through it goes wrong in function `make_request` and `response.raise_for_status()`. Specifically:
> 
> ```
> response = requests.request(
>                 method=method, url=url, headers=headers, data=data
>             )
> ```
> 
> It reports that `requests.exceptions.HTTPError: 400 Client Error: Bad Request for url: https://eval.ai/api/jobs/challenge/2253/update_submission/` and the content in the website is `{"detail":"Authentication credentials were not provided."}`
> 
> How can I fix it?

same issue，do you have any solution?

**Comment by Espere-1119-Song** [comment]
Sorry, I didn't fix it. It seems I need to docker-compose up --build first, but I failed.

**Comment by Liuziyu77** [comment]
> Sorry, I didn't fix it. It seems I need to docker-compose up --build first, but I failed.

Hey, I found a solution which might be useful. I changed my output format as following, and the problem is fixed.
`https://github.com/Cloud-CV/EvalAI-Starters/blob/master/remote_challenge_evaluation/evaluate.py#L52`

**Comment by Espere-1119-Song** [comment]
Did you install environments with docker-compose?

**Comment by Liuziyu77** [comment]
> Did you install environments with docker-compose?

No. I run `./remote_challenge_evaluation/main.py` locally in my conda env to hold a remote evaluation.
